containers/Home/Language/Language.tsx

```diff
@@ -0,0 +1,31 @@
+import {
+    StyledLanguage,
+    StlyedProcessRing,
+    StyledWrapperProcessRing,
+    StlyedTitleProcessRing,
+} from './styles'
+import { BackGroundColor, FontColor, GridBreakpoints } from 'style/Theme'
+
+import { LanguageProps, Prop } from './interface'
+
+export const Language: React.FC<Prop> = ({ languages }) => {
+    return (
+        <StyledLanguage>
+            {languages.map((item, index) => (
+                <div key={index}>
```

Comment:
Can we move the `key={index}` to `StyledWrapperProcessRing` and remove the `div` ?